### ISSUE TITLE
The location of the `texstudio.ini` file should be documented in the documentation

### ISSUE THREAD

**Issue by dbitouze** [texstudio-org/texstudio#3670]
### Describe the feature and the current behavior/state
The location of the `texstudio.ini` file, AFAIK:

- Linux/Unix/Mac: `~/.config/texstudio/texstudio.ini`
- Windows: `%APPDATA%\texstudio\texstudio.ini`

should be documented in the documentation.

### Who will benefit with this feature?
Everyone.

**Comment by sunderme** [comment]
Like here https://github.com/texstudio-org/texstudio/wiki/Frequently-Asked-Questions#where-are-the-settings-stored ?

**Comment by dbitouze** [comment]
Indeed, the wiki is probably not the first place a user would look for such information.

**Comment by octaeder** [comment]
The .ini file contains information that a user should not think about or even edit. I suggest adding a small chapter to the manual say "further reading" or so. I would place it before chapter CHANGELOG. Of course we should not maintain duplicates.
